public/redux/reducers/ad.ts

```diff
@@ -94,7 +94,7 @@ const reducer = handleActions<Detectors>(
       FAILURE: (state: Detectors, action: APIErrorAction): Detectors => ({
         ...state,
         requesting: false,
-        errorMessage: action.error.data.error,
+        errorMessage: action.error,
```

Comment:
This line here was causing an exception. `data` was undefined, which caused the error to be thrown. Because of this, the ad state never got out of `requesting` and was preventing the detector details pages from handling the error and redirecting to the detector list page.